src/Pmi/Cache/DatastoreAdapter.php

```diff
@@ -52,6 +52,7 @@ protected function doDelete(array $ids)
 
     protected function doSave(array $values, $lifetime)
     {
+        $failed = []; // Overwritten in method below (passed by reference)
```

Comment:
Not sure how the Symfony library itself gets away with this one, but this made sense to me to squelch the item.

https://github.com/symfony/symfony/blob/fbd8d8ecb739724747ab31828e699e30d748de51/src/Symfony/Component/Cache/Adapter/CouchbaseBucketAdapter.php#L224-L242